root/artist/ArtistMerge.js

```diff
@@ -65,9 +65,23 @@ const ArtistMerge = ({$c, form, toMerge}: Props) => {
           <FormRowCheckbox
             field={form.field.rename}
             label={l(`Update matching artist and relationship credits to use
-                      the new artist’s name`)}
+                      the target artist’s name`)}
           />
 
+          <p>
+            {exp.l(
+              `By default, the merge will combine the artists but the previous
+              name(s) will still be used on the {doc_acs|artist credits}. If
+              all the name(s) being merged are incorrect (e.g. typos) and all
```

Comment:
```suggestion
               all the names being merged are incorrect (e.g. typos) and all
```
since names can't be singular due to "all," plus "names" being used later in the sentence

(lines needs to be indented an extra space to line up with the first, too)